exercises/go-2-a/.meta/start_2.go

```diff
@@ -0,0 +1,31 @@
+package go2a
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var cutoffs = []struct {
+	amount      int
+	description string
+}{
+	{144, "gross"},
+	{20, "score"},
+	{12, "dozen"},
+}
+
+func Describe(amount int) string {
+	descriptions := make([]string, 0)
+
```

Comment:
```suggestion
```

Reply:
In Go variables are not declared at the beginning of a function. They are declared when needed. Declaring a slice before a loop filling it is a typical pattern we would use. Keeping that as a block makes more sense to me.